src/Symfony/Component/Validator/ConstraintViolationList.php

```diff
@@ -198,4 +198,14 @@ public function findByCodes($codes)
 
         return new static($violations);
     }
+
+    /**
+     * Returns whether the violation list is empty.
+     *
+     * @return bool
+     */
+    public function isEmpty()
+    {
+        return count($this->violations) === 0;
```

Comment:
We don't need to call `count()` anymore.

```suggestion
        return [] === $this->violations;
```

Reply:
Switched to your suggested approach.